src/devtools.ts

```diff
@@ -50,9 +50,10 @@ export function preactDevtoolsPlugin({
 				code = `import "${source}";\n${code}`;
 
 				log(`[inject] ${kl.cyan(source)} -> ${kl.dim(id)}`);
+				return code;
 			}
 
-			return code;
+			return;
```

Comment:
Close, I mean we can drop this line completely ;)

```suggestion
```

Reply:
oops, you could give me at least 5 minutes before merging :)